src/pal/src/include/pal/context.h

```diff
@@ -375,6 +375,59 @@ const fpsimd_context* GetConstNativeSigSimdContext(const native_context_t *mc)
 #define MCREG_Pc(mc)        ((mc).arm_pc)
 #define MCREG_Cpsr(mc)      ((mc).arm_cpsr)
 
+
+// Flatterned layout of the arm kernel struct vfp_sigframe
+struct VfpSigFrame
+{
+    DWORD   magic;
+    DWORD   size;
+    DWORD64 D[32];
```

Comment:
From the linux kernel (5.1.15) we have: 

> User specific VFP registers. If only VFPv2 is present, registers 16 to 31
> are ignored by the ptrace system call and the signal handler.

Is this worth mentioning here?

Reply:
```suggestion
    DWORD64 D[32]; // Some arm cpus only have 16 D registers.  The kernel structure is invariant.  The extra registers will be ignored on these machines
```